autorest/adal/cli.go

```diff
@@ -1,16 +1,18 @@
 package adal
```

Comment:
I thought about this a little this morning, and think that this code would be better off in a `cli` package. Particularly, I think it makes sense as a child of the [`azure` package](https://github.com/Azure/go-autorest/tree/5432abe734f8d95c78340cd56712f912906e6514/autorest/azure). In general, I'd like to have as few Azure specific references in the `adal` package as possible, that applies doubly to CLI specific code :)

Reply:
updated :)